src/ban.h

```diff
@@ -20,15 +20,18 @@
 #ifndef FS_BAN_H_CADB975222D745F0BDA12D982F1006E3
 #define FS_BAN_H_CADB975222D745F0BDA12D982F1006E3
 
-struct BanInfo {
+struct BanInfo
+{
 	std::string bannedBy;
 	std::string reason;
 	time_t expiresAt;
 };
 
-struct ConnectBlock {
-	constexpr ConnectBlock(uint64_t lastAttempt, uint64_t blockTime, uint32_t count) :
-		lastAttempt(lastAttempt), blockTime(blockTime), count(count) {}
+struct ConnectBlock
+{
+	constexpr ConnectBlock(uint64_t lastAttempt, uint64_t blockTime, uint32_t count) : lastAttempt(lastAttempt), blockTime(blockTime), count(count)
+	{
+	}
```

Comment:
I mean, it should be `constexpr ConnectBlock(uint64_t lastAttempt, uint64_t blockTime, uint32_t count) : lastAttempt(lastAttempt), blockTime(blockTime), count(count) {}` on a single line or `{}` on the next line.